gubernator/templates/filtered_log.html

```diff
@@ -12,7 +12,8 @@
 		</h3>
 		<input type="hidden" name="junit" value="{{junit}}">
 		<hr>
-		<label><input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping on</label>
+		<label><input type="checkbox" name="wrap"{% if wrap %} checked{% endif %}> Line wrapping </label>
+		<label><input type="checkbox" name="weave"{% if weave %} checked{% endif %}> Weaving by timestamp </label>
```

Comment:
nit: UI generally uses the imperative tense (e.g. `Wrap lines` or `Weave by timestamp`)